### ISSUE TITLE
Find way to avoid wrapping dates in tables in mobile view

### ISSUE THREAD

**Issue by zkamvar** [carpentries/varnish#80]
Not sure how to do this because [dates are incredibly hard to parse](https://zkamvar.netlify.app/blog/2020-01-02-the-dating-game/), but there is a situation that was found where dates wrap in mobile view: https://github.com/carpentries/lesson-transition/issues/65

![partial screenshot of mobile view where a table containing ISO 8601 formatted dates are wrapped to the day](https://user-images.githubusercontent.com/3639446/234632934-91240ac0-70c9-4f7a-8ec2-5c0bc18fe0f6.png)

**Comment by bencomp** [comment]
Did you fix this already? Looking at this table in Firefox's responsive mode, the issue is not very prominent anymore.
My suggestion would be to use more of the screen when space is tight. Set smaller margins and a smaller gap between the columns. It will not work for all tables, but it should work for this one.

**Comment by bencomp** [comment]
> It's weirdly difficult to define how wide a table should be using markdown: https://pandoc.org/MANUAL.html#tables

I should have clarified that I meant this be changed in the CSS. Looking at your screenshot I feel the gap between the columns is quite big, as well as the margins on the left and right of the table. If your phone is cute/small, better make good use of the available space!
This would not rule out setting `overflow-wrap: word` as well.
